packages/tool-cache/src/tool-cache.ts

```diff
@@ -533,10 +528,28 @@ function _evaluateVersions(versions: string[], versionSpec: string): string {
   return version
 }
 
+/**
+ * Gets RUNNER_TOOL_CACHE
+ */
+function _getCacheDirectory(): string {
+  const cacheDirectory = process.env['RUNNER_TOOL_CACHE'] || ''
```

Comment:
Thoughts on defaulting to `.`?
`Path.join` ignores empty strings, it should be the same result, but using `.` means these functions do return valid paths.
Same for below.

Reply:
i added `|| ''` to satisfy typescript. The `ok` check verifies not empty.